### PR TITLE
Contributor: Create spryker-cloud-commerce-management.md

### PR DESCRIPTION
**Pull request by VadymSachenko**
## PR Description
Original PRs: 
- https://github.com/spryker/spryker-docs/pull/1284
- https://github.com/spryker/spryker-docs/pull/1367 — changes added manually

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: _data/sidebars/cloud_dev_sidebar.yml [hunk 1]
```diff
@@ -131,3 +131,5 @@ entries:
                 url: /docs/cloud/dev/spryker-cloud-commerce-os/troubleshooting/troubleshooting-tutorials/tutorial-troubleshooting-yves-or-zed-not-restarting-after-deployment.html
       - title: Release notes
         url: /docs/cloud/dev/spryker-cloud-commerce-os/release-notes.html
+      - title: Spryker Cloud Commerce OS management
+      - url: /docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.html
```
Comment (VadymSachenko) [inline _data/sidebars/cloud_dev_sidebar.yml:135]:
```suggestion
      url: /docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.html
```

**Comment by fscholzspryker** [comment]
Related to the other PR I also approve the changes in docs/scos/dev/back-end-development/data-manipulation/data-publishing/handle-data-with-publish-and-synchronization.md

File: docs/scos/dev/back-end-development/data-manipulation/data-publishing/handle-data-with-publish-and-synchronization.md [hunk 2]
```diff
@@ -706,7 +716,88 @@ class HelloWorldStorageFacade extends AbstractFacade implements HelloWorldStorag
 }
 ```
 
-4. Refactor the publisher classes and call the Facade methods:
+4. In order to connect the facade methods to the business logic in the Writer and Deleter, we need to create the Business factory that creates the Writer abd Deleter objects. It is also good practice to create interfaces for these objects.
```
Comment (andriitserkovnyi) [inline docs/scos/dev/back-end-development/data-manipulation/data-publishing/handle-data-with-publish-and-synchronization.md:719]:
```suggestion
4. To connect the facade methods to the business logic in the Writer and Deleter, create the Business factory that creates the Writer and Deleter objects. We also recommend creating interfaces for these objects.
```

File: docs/scos/dev/back-end-development/data-manipulation/data-publishing/handle-data-with-publish-and-synchronization.md [hunk 3]
```diff
@@ -706,7 +716,88 @@ class HelloWorldStorageFacade extends AbstractFacade implements HelloWorldStorag
...
+
+Create the file: `src\Pyz\Zed\HelloWorldStorage\Business\HelloWorldStorageBusinessFactory.php`
```
Comment (andriitserkovnyi) [inline docs/scos/dev/back-end-development/data-manipulation/data-publishing/handle-data-with-publish-and-synchronization.md:721]:
```suggestion
Create `src\Pyz\Zed\HelloWorldStorage\Business\HelloWorldStorageBusinessFactory.php`.
```

File: docs/scos/dev/back-end-development/data-manipulation/data-publishing/handle-data-with-publish-and-synchronization.md [hunk 4]
```diff
@@ -706,7 +716,88 @@ class HelloWorldStorageFacade extends AbstractFacade implements HelloWorldStorag
...
+    {
+        return new HelloWorldStorageDeleter();
+    }
+}
+
+```
+
+As you see, these methods return interfaces, so let's create them.
+
+Create the file: `src\Pyz\Zed\HelloWorldStorage\Business\Writer\HelloWorldStorageWriterInterface.php`
```
Comment (andriitserkovnyi) [inline docs/scos/dev/back-end-development/data-manipulation/data-publishing/handle-data-with-publish-and-synchronization.md:761]:
```suggestion
As you see, these methods return interfaces.

In `src\Pyz\Zed\HelloWorldStorage\Business\Writer\HelloWorldStorageWriterInterface.php`, create the interfaces.
```

File: docs/scos/dev/back-end-development/data-manipulation/data-publishing/handle-data-with-publish-and-synchronization.md [hunk 5]
```diff
@@ -706,7 +716,88 @@ class HelloWorldStorageFacade extends AbstractFacade implements HelloWorldStorag
...
+	 * @param   \Generated\Shared\Transfer\EventEntityTransfer[]  $eventTransfers
+	 *
+	 * @return void
+	 */
+	public function writeCollectionByHelloWorldEvents(array $eventTransfers): void;
+}
+
+```
+
+Create the file: `src\Pyz\Zed\HelloWorldStorage\Business\Deleter\HelloWorldStorageDeleterInterface.php`
```
Comment (andriitserkovnyi) [inline docs/scos/dev/back-end-development/data-manipulation/data-publishing/handle-data-with-publish-and-synchronization.md:780]:
```suggestion
Create `src\Pyz\Zed\HelloWorldStorage\Business\Deleter\HelloWorldStorageDeleterInterface.php`.
```

File: docs/scos/dev/back-end-development/data-manipulation/data-publishing/handle-data-with-publish-and-synchronization.md [hunk 6]
```diff
@@ -511,24 +511,31 @@ To synchronize data with Redis, an intermediate Zed database table is required.
 
 Follow the steps to create the table:
 
-1. Create the table schema file:
+1. Create the table schema file in `Pyz\Zed\HelloWorldStorage\Persistance\Propel\Schema\spy_hello_world_storage.schema.xml`:
```
Comment (andriitserkovnyi) [inline docs/scos/dev/back-end-development/data-manipulation/data-publishing/handle-data-with-publish-and-synchronization.md:514]:
```suggestion
1. Create the table schema `Pyz\Zed\HelloWorldStorage\Persistance\Propel\Schema\spy_hello_world_storage.schema.xml`:
```

File: docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
+---
+title: Spryker Cloud Commerce management
```
Comment (andriitserkovnyi) [inline docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md:2]:
```suggestion
title: Spryker Cloud Commerce OS management
```

File: docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+The goal of this RACI Matrix is to define and document responsibilities, consultation- and reporting-obligations while providing Platform-as-a-Service (PaaS) hosting for a Spryker customer.
+
+## Glossary
+
+**Responsible**: Those who do the work to complete the task.
+
+**Accountable**: The one ultimately answerable for the correct and thorough completion of the deliverable or task, and the one who delegates the work to those responsible.
+
+**Consulted**: Those whose opinions are sought, typically subject matter experts; and with whom there is two-way communication.
```
Comment (andriitserkovnyi) [inline docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md:16]:
```suggestion
**Consulted**: Those whose opinions are sought, typically subject matter experts; and with whom there is a  two-way communication.
```

File: docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+description: The goal of this RACI Matrix is to define and document responsibilities, consultation- and reporting-obligations while providing Platform-as-a-Service hosting for a Spryker customer.
+last_updated: Nov 21, 2022
+template: concept-topic-template
+---
+
+The goal of this RACI Matrix is to define and document responsibilities, consultation- and reporting-obligations while providing Platform-as-a-Service (PaaS) hosting for a Spryker customer.
+
+## Glossary
+
+**Responsible**: Those who do the work to complete the task.
```
Comment (andriitserkovnyi) [inline docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md:12]:
```suggestion
**Responsible**: Those who do the work to complete a task.
```

File: docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+
+**Accountable**: The one ultimately answerable for the correct and thorough completion of the deliverable or task, and the one who delegates the work to those responsible.
```
Comment (andriitserkovnyi) [inline docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md:14]:
```suggestion
**Accountable**: The one ultimately answerable for the correct and thorough completion of a deliverable or task, and the one who delegates the work to those responsible.
```

File: docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+
+**Consulted**: Those whose opinions are sought, typically subject matter experts; and with whom there is two-way communication.
+
+**Informed**: Those who are kept up-to-date on progress, often only on completion of the task or deliverable; and with whom there is just one-way communication.
```
Comment (andriitserkovnyi) [inline docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md:18]:
```suggestion
**Informed**: Those who are kept up to date on progress, often only on completion of a task or deliverable; and with whom there is just one-way communication.
```

File: docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md [hunk 12]
```diff
@@ -0,0 +1,48 @@
+---
+title: Spryker Cloud Commerce OS management
+description: The goal of this RACI Matrix is to define and document responsibilities, consultation- and reporting-obligations while providing Platform-as-a-Service hosting for a Spryker customer.
```
Comment (andriitserkovnyi) [inline docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md:3]:
```suggestion
description: Assignment of responsibilities, consultation and reporting obligations.
```

File: docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+last_updated: Nov 21, 2022
+template: concept-topic-template
+---
+
+The goal of this RACI Matrix is to define and document responsibilities, consultation- and reporting-obligations while providing Platform-as-a-Service (PaaS) hosting for a Spryker customer.
```
Comment (andriitserkovnyi) [inline docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md:8]:
```suggestion
This RACI Matrix defines responsibilities, consultation and reporting obligations while providing Platform-as-a-Service (PaaS) hosting for Spryker customers.
```

File: docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+**Responsible**: Those who do the work to complete the task.
+
+**Accountable**: The one ultimately answerable for the correct and thorough completion of the deliverable or task, and the one who delegates the work to those responsible.
+
+**Consulted**: Those whose opinions are sought, typically subject matter experts; and with whom there is two-way communication.
+
+**Informed**: Those who are kept up-to-date on progress, often only on completion of the task or deliverable; and with whom there is just one-way communication.
+
+
+**Application**: Customer Commerce Solution build using Spryker Commerce OS.
```
Comment (andriitserkovnyi) [inline docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md:21]: @michalwroblewski-spryker what is a `Customer Commerce Solution`?

File: docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md [hunk 15]
```diff
@@ -0,0 +1,48 @@
+---
+title: Spryker Cloud Commerce management
+description: The goal of this RACI Matrix is to define and document responsibilities, consultation- and reporting-obligations while providing Platform-as-a-Service hosting for a Spryker customer.
+last_updated: Nov 21, 2022
+template: concept-topic-template
+---
+
+The goal of this RACI Matrix is to define and document responsibilities, consultation- and reporting-obligations while providing Platform-as-a-Service (PaaS) hosting for a Spryker customer.
+
+## Glossary
```
[the remark this answers is not in the document]
Comment (VadymSachenko) [reply docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md:10]: This format uses HTML tags. I think we should follow this formatting as close as possible through markdown. What do you think? 
Or is it a rule I wasn't aware of before and we must use HTML for glossaries? @andriitserkovnyi

[the remark this answers is not in the document]
Comment (andriitserkovnyi) [reply docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md:10]: As far as I remember, we actually made a frontend dev create a custom css to style glossaries. So yeah, we have to use html, but we have few glossaries, so it's not much work.

File: docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md [hunk 16]
```diff
@@ -0,0 +1,48 @@
...
+**Accountable**: The one ultimately answerable for the correct and thorough completion of a deliverable or task, and the one who delegates the work to those responsible.
+
+**Consulted**: Those whose opinions are sought, typically subject matter experts; and with whom there is a  two-way communication.
+
+**Informed**: Those who are kept up to date on progress, often only on completion of a task or deliverable; and with whom there is just one-way communication.
+
+
+**Application**: Customer Commerce Solution build using Spryker Commerce OS.
+
+**Infrastructure**: Underlying cloud infrastructure managed by Spryker to deliver Spryker Cloud Commerce to Spryker customers.
```
Comment (andriitserkovnyi) [inline docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md:23]:
```suggestion
**Infrastructure**: Underlying cloud infrastructure managed by Spryker to deliver Spryker Cloud Commerce OS to customers.
```

File: docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md [hunk 17]
```diff
@@ -0,0 +1,48 @@
...
+
+## RACI Matrix
+
+| ACTIVITY | CUSTOMER | RESPONSIBLE DEVELOPING PARTY (CUSTOMER OR PARTNER) | SPRYKER |
+| --- | ---| --- | --- |
+| Application Health, Security and Performance|
+| Provide code operational application (docker-sdk and Spryker Code) | C | A/R | I |
```
Comment (andriitserkovnyi) [inline docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md:30]: What is a `code operational application`? @michalwroblewski-spryker

File: docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md [hunk 18]
```diff
@@ -0,0 +1,48 @@
...
+**Informed**: Those who are kept up to date on progress, often only on completion of a task or deliverable; and with whom there is just one-way communication.
+
+
+**Application**: Customer Commerce Solution build using Spryker Commerce OS.
+
+**Infrastructure**: Underlying cloud infrastructure managed by Spryker to deliver Spryker Cloud Commerce to Spryker customers.
+
+## RACI Matrix
+
+| ACTIVITY | CUSTOMER | RESPONSIBLE DEVELOPING PARTY (CUSTOMER OR PARTNER) | SPRYKER |
```
Comment (andriitserkovnyi) [inline docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md:27]: @michalwroblewski-spryker what do blank fields mean when opposed to `-`?

File: docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md [hunk 19]
```diff
@@ -0,0 +1,48 @@
...
+| Provide code operational application (docker-sdk and Spryker Code) | C | A/R | I | 
+| Optimize Application  performance applying best practices | I/C | R/A | I | 
+| Maintain Application up-to-date and secure | I/C | R/A | I |
+| Infrastructure Configuration and Scaling | 
+| Manage environment sizing depending on the current needs | - | C | R/A |
+| Maintain the Infrastructure up-to-date and secure | I | I | R/A |
+| Configuration of the infrastructure for the Application | I | C | R/A |
+| Provide deploy file that can be used in the Spryker Cloud setup | I | R/A | C |
+| Deployments |
+| Provide valid access tokens for the repositories and the repository is available to Spryker Cloud as per requirements | - | R/A | I |
```
Comment (andriitserkovnyi) [inline docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md:39]: What is `available to Spryker Cloud` mean?

File: docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md [hunk 20]
```diff
@@ -0,0 +1,48 @@
...
+| Conduct appropriate (load) testing, especially regarding data imports before production deployment | I/C | R/A | I |
```
Comment (andriitserkovnyi) [inline docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md:40]: @michalwroblewski-spryker Is it just load testing or load testing and some other testing?

File: docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md [hunk 21]
```diff
@@ -0,0 +1,57 @@
+---
+title: Spryker Cloud Commerce OS management
+description: Assignment of responsibilities, consultation and reporting obligations.
+last_updated: Nov 21, 2022
+template: concept-topic-template
+---
+
+This RACI Matrix defines responsibilities, consultation and reporting obligations while providing Platform-as-a-Service (PaaS) hosting for Spryker customers.
+
+<div class="bg-section">
```
Comment (VadymSachenko) [inline docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md:10]: @andriitserkovnyi I've changes Glossary according to the structure you've shared

File: docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md (hunk 14 above)
Comment (michalwroblewski-spryker) [reply docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md]: @andriitserkovnyi  This is the commerce solution the customer build based on our Spryker. Eg by customizing the existing b2b demo shop.

File: docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md (hunk 17 above)
Comment (michalwroblewski-spryker) [reply docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md:39]: It means Spryker Commerce implementation that works.

File: docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md (hunk 18 above)
Comment (michalwroblewski-spryker) [reply docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md:36]: @andriitserkovnyi it is explained in the glossary section

File: docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md (hunk 20 above)
Comment (michalwroblewski-spryker) [reply docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md:49]: @andriitserkovnyi it is especially load tests but also following industry best practices in testing the solution before deployement

**Review by andriitserkovnyi — APPROVED** [review]

**Pull request merged**